conda_smithy/configure_feedstock.py

```diff
@@ -390,7 +390,7 @@ def main(forge_file_directory):
               'circle': {},
               'appveyor': {},
               'channels': {'sources': ['conda-forge'], 'targets': [['conda-forge', 'main']]},
-              'github': {'user_or_org': 'conda-forge', 'repo_name': ''},
+              'github': {'user_or_org': 'conda-forge', 'repo_name': '', 'branch_name': 'master'},
```

Comment:
I fully support your usecase and would like the same to be true when you look at the README of https://github.com/conda-forge/python-feedstock/tree/2.7. I certainly want to be careful though - the name of a branch should not live in the repo itself (otherwise you end up having to make a code change just to take a branch). Some thought will be needed to figure out what heuristic we can reasonably depend upon to determine the branch - let's spin the conversation into its own issue (though I can't promise to be able to commit any time to it in the next few weeks due to ✈️  -> 🐍  🌅  🐨  ).